config/initializers/highlight_override.rb

```diff
@@ -0,0 +1,54 @@
+# Monkey patch the generated Bindings for the swagger Highlight model
+
+Api::V0::Bindings::NewHighlight.class_exec do
+  def valid_location_strategies?
+    location_strategies.present? &&
```

Comment:
The binding already checks against `location_strategies.nil?`

Reply:
present checks for both nil? and empty array (which is relevant here...if the location strategies are [])